pkg/web/static/js/countriesList.js

```diff
@@ -0,0 +1,247 @@
+export const countries = {
```

Comment:
This object has been moved to a separate file so that it may be imported in other `js` files.